src/PrivateDns/PrivateDns/Zones/SetAzurePrivateDnsZone.cs

```diff
@@ -94,7 +94,10 @@ public override void ExecuteCmdlet()
                 throw new PSArgumentException(string.Format(ProjectResources.Error_ZoneNotFound), this.Name);
             }
 
-            zoneToUpdate.Tags = this.Tag;
+            if (this.Tag != null)
```

Comment:
Same comment - this should not be reverted.

Reply:
same question as in the above comment. what if the user decides to update tags on the local powershell object instead of passing Tags as a param?

Reply:
I would recommend changing this to "Update" if you want to support that behavior.

Reply:
not changing this to update, i'll revert this change instead